server/app/app.js

```diff
@@ -55,14 +56,43 @@ const authRoutes = require("./routes/authRoutes");
 const signpostRoute = require("./routes/signpostRoute");
 const userRoutes = require("./routes/userRoute");
 const milestoneRoute = require("./routes/milestone");
-const CodewarsRoutes = require("./routes/codewarsRoutes")
+const CodewarsRoutes = require("./routes/codewarsRoutes");
+const ReposService = require("./helpers/ReposService");
 
 app.use("/auth", authRoutes);
 app.use("/signpost", signpostRoute);
 app.use("/user", userRoutes);
 app.use("/current-milestone", milestoneRoute);
-app.use("/codewars", CodewarsRoutes )
+app.use("/codewars", CodewarsRoutes);
+
+app.get("/getAllRepos/:GithubAccount", async (req, res) => {
```

Comment:
Do we need to pass `GithubAccount` param? Can we get it from the authenticated user?

Reply:
I think on the backend we need to get GithubAccount. what do you think?

Reply:
No, the backend already knows trainee's githib username because a trainee must be authenticated to see this page.